### PR TITLE
Adding tests

### PR DESCRIPTION
**Pull request by MorganLove**
**Testing**

- Filling in the gaps

### CODE REVIEW

**Comment by MorganLove** [comment]
These tests are being oddly flaky.

File: client/src/components/DataEntry/SignOff.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const SignOff = ({ auditBoard, submitSignoff }: IProps) => {
                 label={`Audit Board Member: ${member.name}`}
               >
                 <LabelText htmlFor={`[${i}]`}>Full Name</LabelText>
-                <NameField name={`[${i}]`} />
+                <NameField name={`[${i}]`} id={`[${i}]`} />
```
Comment (jonahkagan) [inline client/src/components/DataEntry/SignOff.tsx:40]: why this change?

File: client/cypress/end-to-end/ballot-comparison-failure-cases.spec.js [hunk 2]
```diff
@@ -217,6 +217,13 @@ describe('Ballot Comparison Failure Cases', () => {
     cy.loginAuditAdmin(auditAdmin)
     cy.findByText(`TestAudit${id}`).click()
     cy.findByText('Review & Launch').click()
+  
+    // custom size validation test
+    cy.findByText('Enter your own sample size (not recommended)').click()
+    cy.findByRole('spinbutton').type('30').blur()
+    cy.findByText('Must be less than or equal to: 15 (the total number of ballots in the contest)')
```
Comment (jonahkagan) [inline client/cypress/end-to-end/ballot-comparison-failure-cases.spec.js:224]: this is frontend validation, right? if so, i think this would be better tested in a unit test, not an end-to-end test, since it doesn't involve communication between front and backend

File: client/src/components/DataEntry/SignOff.tsx (hunk 1 above)
Comment (MorganLove) [reply client/src/components/DataEntry/SignOff.tsx:40]: @jonahkagan Needed to find the input field of 'Full Name' by label, and the reference of the label wasn’t returning anything because the id of the input was not available.

**Review by jonahkagan — APPROVED** [review]
yay for more tests!
